code-reviews/recipes/Markdown.md

```diff
@@ -85,17 +85,19 @@ npx write-good *.md
 
 Write Good is also available as an [extension for VS Code](https://marketplace.visualstudio.com/items?itemName=travisthetechie.write-good-linter)
 
-## VS Code Extensions
+### VS Code Extensions
```

Comment:
This should stay a level 2 title to follow the same format as the other recipes

Reply:
I have got impression by reading the document that we are talking about linting tools here and VS code extensions are one of those tools - hence the change to level of title. Did I mistunderstood? @TessFerrandez

Reply:
I think we should be consistent between recipes, the VS code extensions could be linting tools, they could also be other extensions that are useful - e.g. viewers, or in this case the write good extension etc. that are not necessarily linters

Reply:
Yes, that sound good then,  updated.